File: Herencia/ejercicio 1/Python/Coche.py
from Vehiculo import Vehiculo
class Coche(Vehiculo):
    def __init__(self, marca, modelo, año, precio, nPuertas, tConbustible):
        super().__init__(marca, modelo, año, precio)
        self.nPuertas = nPuertas
        self.tConbustible = tConbustible
    def mostrar(self):
        return super().mostrar(), "Puertas: ", self.nPuertas, "Combustible: ", self.tConbustible

File: Herencia/ejercicio 1/Java/Coche.java
public class Coche extends Vehiculo{
    public int puerta;
    public String conbustible;

    public Coche(String marca, String modelo, int año, double precio, int puerta, String conbustible) {
        super(marca, modelo, año, precio);
        this.puerta = puerta;
        this.conbustible = conbustible;
    }

    public String mostrar() {
        return super.mostrar() + "puerta: " + puerta + "conbustible: " + conbustible;
    }
    
    public int getnPuertas() {
        return puerta;
    }
}
